code/modules/mob/living/carbon/update_icons.dm

```diff
@@ -125,6 +125,13 @@
 	if(wear_mask)
 		if(!(head && (head.flags_inv & HIDEMASK)))
 			overlays_standing[FACEMASK_LAYER] = wear_mask.build_worn_icon(state = wear_mask.icon_state, default_layer = FACEMASK_LAYER, default_icon_file = 'icons/mob/mask.dmi')
+			if(ishuman(src))

```

Comment:
No. If it's human functionality you do it on human level.

Reply:
sure thing boss